docker/docker-compose.yml

```diff
@@ -0,0 +1,44 @@
+version: '3.0'
+
+services:
+  rasacore:
```

Comment:
i think we should stick to one naming convention (`action_serve` vs `rasacore`), i guess this should be `rasa_core`